### PR TITLE
Add reference mapping to docker commands - #305

### PR DESCRIPTION
**Pull request by SydOps**
When read the usage, I do find it is hard to get quick understanding to mapping the ruby gem with real docker commands. 

Add comments in README for a quick start. If you like this way, I will add the rest. 

Second, when do the sample to pull image, the command use a fake image named `base`, change it to a real and tiny one, developer can quickly understand how to use it properly

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,8 +123,9 @@ Just about every method here has a one-to-one mapping with the [Images](https://
 require 'docker'
 # => true
 
-# Create an Image.
-image = Docker::Image.create('fromImage' => 'base')
+# Pull an Image.
+# docker command for reference: docker pull alpine:3.2
+image = Docker::Image.create('fromImage' => 'alpine:3.2')
```
Comment (tlunter) [inline README.md]: Can you make this ubuntu:14.04?

**Comment by tlunter** [comment]
@SydOps looks good, just would prefer ubuntu since that's what base used to be. Will merge when changed!

**Comment by SydOps** [comment]
updated.

**Comment by tlunter** [comment]
Thanks a bunch!

**Pull request merged**